addons/resource.language.en_gb/resources/strings.po

```diff
@@ -10976,11 +10976,11 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#20369"
-msgid "Show plot for unwatched items"
+msgid "Show plot for unwatched items (Hides plot and thumb when disabled)"
```

Comment:
Thats fine if people read the description text, I dont mind and the shorter the better.